assert/assertions_test.go

```diff
@@ -2189,3 +2190,22 @@ func Test_validateEqualArgs(t *testing.T) {
 		t.Error("nil functions are equal")
 	}
 }
+
+func Test_truncatingFormat(t *testing.T) {
+
+	original := strings.Repeat("a", bufio.MaxScanTokenSize-102)
+	result := truncatingFormat(original)
+	if fmt.Sprintf("%#v", original) != result {
```

Comment:
Excuse me, I forgot to point out that just like you can use `Equal` below, you can use `NotEqual` here.